### ISSUE TITLE
Group participant name

### ISSUE THREAD

**Issue by Lucipher** [orkestral/venom#616]
Is there any method to get participant name (the same as getGroupMembers's pushname) with result of function  getAllGroups  return?

**Comment by Lucipher** [comment]
"participants" => [
        [
            "id" =>  [
              "server" => "c.us",
              "user" => "888888",
              "_serialized" => "888888@c.us"
            ],
            "isAdmin" => false,
            "isSuperAdmin" =>  false,
            **+pusname** 
          ]
     ]

**Comment by jonalan7** [comment]
**if you want information about the members, just do this:**
 _This example returns only one group._
```
    const groups = await client.getAllChatsGroups();
    const members = await client.getGroupMembers(groups[0].id._serialized);
    console.log("Get Members: ", members);
```

**Comment by orkestral** [comment]
fixed in next version 3.0.6, thank you!
